docs/src/python/dlt.md

```diff
@@ -0,0 +1,142 @@
+
+
+3. **Specify necessary credentials and/or embedding model details:**  
+    
+    In order to fetch data from the OMDb API, you will need to pass a valid API key into your pipeline. Additionally, depending on whether you're using LanceDB OSS or LanceDB cloud, you also may or may not need to provide necessary credentials to connect to the LanceDB instance. These can be pasted inside `.dlt/sercrets.toml`. 
+
+    Furthermore, `dlt`'s LanceDB integration allows you to automatically embed the data during ingestion. Again, depending on the embedding model chosen, you may need to paste necessary credentials inside `.dlt/sercrets.toml`:
+    ```toml
+    [sources.rest_api]
+    api_key = "api_key" # Enter the API key for the OMDb API
+
+    [destination.lancedb]
+    embedding_model_provider = "sentence-transformers"
+    embedding_model = "all-MiniLM-L6-v2"
+    [destination.lancedb.credentials]
+    uri = ".lancedb"
+    api_key = "api_key" # API key to connect to LanceDB Cloud. Leave out if you are using LanceDB OSS.
+    embedding_model_provider_api_key = "embedding_model_provider_api_key" # Not needed for providers that don't need authentication (ollama, sentence-transformers).
+    ```
+    See [here](https://dlthub.com/docs/dlt-ecosystem/destinations/lancedb#configure-the-destination) for more information and for a list of available models and model providers.  
+
+
+4. **Write the pipeline code inside `rest_api_pipeline.py`:**  
+
+    The following code shows how you can configure `dlt`'s REST API source to connect to the [OMDb API](https://www.omdbapi.com/), fetch all movies with the word "godzilla" in the title, and load it into a LanceDB table. For more details on the `dlt` REST API source, see [here](https://dlthub.com/docs/dlt-ecosystem/verified-sources/rest_api).
+
+    ```python   
+
+    # Import necessary modules
+    import dlt
+    from rest_api import rest_api_source
+
+    # Configure the REST API source
+    movies_source = rest_api_source(
+        {
+            "client": {
+                "base_url": "https://www.omdbapi.com/",
+                "auth": { # authentication strategy for the OMDb API
+                    "type": "api_key",
+                    "name": "apikey",
+                    "api_key": dlt.secrets["sources.rest_api.api_token"], # read API credentials directly from secrets.toml
+                    "location": "query"
+                },
+                "paginator": { # pagination strategy for the OMDb API 
+                    "type": "page_number",
+                    "base_page": 1,
+                    "total_path": "totalResults",
+                    "maximum_page": 5
```

Comment:
is this a paginator that dlt can't detect automatically?

Reply:
Yeah, you would need to explicitly specify it for this API